libs/sdk-ui-kit/src/SettingWidget/tests/StylingSettingWidget.test.tsx

```diff
@@ -122,6 +122,18 @@ describe("StylingPicker", () => {
         expect(screen.queryByLabelText("second_theme")).not.toBeChecked();
     });
 
+    it("should callable onCancel when disabledCancelButton is false and cancel button is clicked", async () => {
```

Comment:
Please change `callable` → `call` to be somewhat more grammatically correct and to be consistent with the description at line 137 below.

Reply:
Also, if we rename the prop's name to `shouldDisableCancelButton`, don't forget to update the name here accordingly.